S21/osamaibraheem11/s21.py

```diff
@@ -0,0 +1,26 @@
+import csv
+
+vowels = ["A","E","I","O","U","Á","É","Í","Ó","Ú"]
+
+def LowerCase(word,language): 
+  if ('ga' in language) and (word.startswith('n') or word.startswith('t')) and word[1:2] in vowels:
+    return word[:1].lower()+"-"+word[1:].lower()
+  if 'en' in language:
+    return word.lower()
+  if ('tr' or "az" in language):
+    return word.replace('I',"ı").lower()
+  if ('el' in language):
+    if word.endswith('Σ'):
```

Comment:
I'm worried about a word that ends with Σ, but also has other Σ's earlier in the word.  Since replace is global, won't those also be lowercased to ς?